resources/views/layout.blade.php

```diff
@@ -24,56 +26,53 @@
                     <div class="nav-item me-2">
                         <a class="nav-link text-uppercase" href="{{Route('home')}}">home</a>
                     </div>
+                    <div class="nav-item me-2">
+                        <a class="nav-link text-uppercase" href="{{Route('listings.index')}}">Advertenties</a>
+                    </div>
                     <div class="ms-auto d-flex">
                         <div class="nav-item me-2">
                             <a class="btn btn-primary" href="{{Route('listings.create')}}">Plaats Advertentie</a>
                         </div>
```

Comment:
```suggestion
                    <div class="ms-auto d-flex">
                        <div class="nav-item me-2">
                            <a class="btn btn-primary" href="{{Route('listings.create')}}">Plaats Advertentie</a>
                        </div>
```

Het is verstandig dat het check wordt dat gebruiker een adverteerder is, want nu kan de klant een advertentie plaatsen